pkg/cloud/alibaba/provider_test.go

```diff
@@ -112,6 +112,20 @@ func TestProcessDescribePriceAndCreateAlibabaPricing(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		{
+			name: "test General Purpose Type g8a instance family",
+			teststruct: &SlimK8sNode{
+				InstanceType:       "ecs.g8a.8xlarge",
+				RegionID:           "cn-hangzhou",
+				PriceUnit:          "Hour",
+				MemorySizeInKiB:    "33554432KiB",
+				IsIoOptimized:      true,
+				OSType:             "Linux",
+				ProviderID:         "cn-hangzhou.i-test-01c",
+				InstanceTypeFamily: "g8a",
+			},
+			expectedError: nil,
+		},
```

Comment:
@qixiangyang, I'm concerned that simple checking no error won't ensure that this change works as you expect it to. 

Would you be able to enhance the test case to take in a literal request.CommonRequest that you can then do a deep equal on? Or even just a map of the expected query params. That way, this test would ensure your logic is actually applied in this case